services/core-api/app/api/notice_of_departure/models/notice_of_departure.py

```diff
@@ -9,21 +9,45 @@
 from app.api.utils.include.user_info import User
 from sqlalchemy.sql import text, select, table, column, literal_column
 from sqlalchemy.sql.functions import func
+from sqlalchemy import desc, asc
 
 
 class NodType(Enum):
-    non_substantial = auto()
-    potentially_substantial = auto()
+    non_substantial = "non_substantial"
```

Comment:
I'm not too familiar with this syntax.  Does this just mean that the default value if none is provided will be `non_substantial`?